app/models/client.rb

```diff
@@ -368,85 +368,6 @@ def active_in_match?
     active_matches.any?
   end
 
-  def available_as_candidate_for_any_route?
```

Comment:
This was moved into the `Availability` concern, where similar methods already existed.